picatic.io/app/services/ExamplesService.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.factory('ExamplesService', [ function() {
+    var ExamplesService = {
+      examples: [
+        // {
```

Comment:
Try not to leave "commented out" blocks in public code.
You can always look back at an older commit to see what you had there if needed.